documentation/0.13.0/html/proxy-quick-start/index.markdown

```diff
@@ -152,3 +152,117 @@ kaf -b $KROXYLICIOUS_BOOTSTRAP consume my_topic
 [//]: # (====================================================================)
 [//]: # (END - Use Section Examples Tabbed Card)
 [//]: # (====================================================================)
+
+# Running Kroxylicious via Docker Compose
+
+## Prerequisites
+
+Make sure to have either [Docker Compose](https://docs.docker.com/compose/) or [Podman Compose](https://docs.podman.io/en/stable/markdown/podman-compose.1.html) installed.
+
+## Step 1: Create the Compose file
+
+Create a file named _docker-compose.yaml_ with the following contents:
+
+```yaml
+services:
+  # from https://github.com/apache/kafka/blob/trunk/docker/examples/docker-compose-files/single-node/plaintext/docker-compose.yml
+  kafka:
+    image: apache/kafka:4.0.0
+    ports:
+      - '9092:9092'
+    environment:
+      KAFKA_NODE_ID: 1
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: 'CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT'
+      KAFKA_ADVERTISED_LISTENERS: 'PLAINTEXT_HOST://localhost:9092,PLAINTEXT://kafka:19092'
+      KAFKA_PROCESS_ROLES: 'broker,controller'
+      KAFKA_CONTROLLER_QUORUM_VOTERS: '1@kafka:29093'
+      KAFKA_LISTENERS: 'CONTROLLER://:29093,PLAINTEXT_HOST://:9092,PLAINTEXT://:19092'
+      KAFKA_INTER_BROKER_LISTENER_NAME: 'PLAINTEXT'
+      KAFKA_CONTROLLER_LISTENER_NAMES: 'CONTROLLER'
+      CLUSTER_ID: '4L6g3nShT-eMCtK--X86sw'
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_SHARE_COORDINATOR_STATE_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_SHARE_COORDINATOR_STATE_TOPIC_MIN_ISR: 1
+      KAFKA_LOG_DIRS: '/tmp/kraft-combined-logs'
+    networks:
+      - my-network
+  kroxylicious:
+    image: quay.io/kroxylicious/kroxylicious:{{ page.version }}
+    ports:
+      - 9192:9192
+      - 9193:9193
+    volumes:
+      - ./proxy-config.yaml:/opt/kroxylicious/config/proxy-config.yaml
+    command: --config=/opt/kroxylicious/config/proxy-config.yaml
+    networks:
+      - my-network
```

Comment:
I ended up playing with compose for an issue and ended up with

```
  kroxylicious:
    image: 'quay.io/kroxylicious/proxy:0.15.0-SNAPSHOT'
    container_name: kroxylicious
    command: --config /opt/kroxylicious/config/proxy-config.yaml
    ports:
      - "9192-9196:9192-9196"
    configs:
      - source: proxy-config
        target: /opt/kroxylicious/config/proxy-config.yaml

configs:
  proxy-config:
    file: ./proxy-config.yaml
```    

The specifying the config/volumes at the top level seemed to help make things work with podman machine on a mac.

I'm not sure what syntax for ports is easier to read either. The one entry per line from your snippet vs the range on each side notation from mine.